### PR TITLE
DDP-4552 . MBC migration related fixes/improvements

### PR DESCRIPTION
**Pull request by ssettipalli**
look at https://broadinstitute.atlassian.net/browse/DDP-4552

### CODE REVIEW

**Comment by ssettipalli** [comment]
please run tests

File: pepper-apis/src/main/resources/db-changes/schema/DDP-4552-modify-medicalprovider-street.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet id="20200422-modify-usermedicalprovider-street" author="sampath">
+
+        <modifyDataType tableName="user_medical_provider" columnName="street" newDataType="varchar(250)"/>
```
Comment (yufengwng) [inline pepper-apis/src/main/resources/db-changes/schema/DDP-4552-modify-medicalprovider-street.xml:9]: Anything unusual about the street that caused this expansion? I'm wondering if we'll need to expand other tables as well, such as the `mailing_address` one.

**Review by yufengwng — DISMISSED** [review]

**Comment by ssettipalli** [comment]
please run tests

**Review by yufengwng — DISMISSED** [review]
Seems fine.

File: pepper-apis/src/main/resources/db-changes/schema/DDP-4552-modify-medicalprovider-street.xml (hunk 1 above)
Comment (ssettipalli) [reply pepper-apis/src/main/resources/db-changes/schema/DDP-4552-modify-medicalprovider-street.xml:9]: mailing_address contains two columns: street1 and street2 with length 100 each. we are good, no issues there.

**Comment by ssettipalli** [comment]
please run tests

**Review by yufengwng — DISMISSED** [review]

**Review by yufengwng — DISMISSED** [review]

**Review by yufengwng — APPROVED** [review]

**Pull request merged**